Zilon.Core/Zilon.Bot.Players.DevelopmentTests/BotActorTaskSourceTests.cs

```diff
@@ -33,6 +37,13 @@ public async Task GetActorTasksTestAsync(string mode)
 
             var scoreManager = _globalServiceProvider.GetRequiredService<IScoreManager>();
             Console.WriteLine($"Scores: {scoreManager.BaseScores}");
+
+            var playerEventLogService = _globalServiceProvider.GetRequiredService<IPlayerEventLogService>();
+            var deathReasonService = _globalServiceProvider.GetRequiredService<DeathReasonService>();
+            var lastEvent = playerEventLogService.GetPlayerEvent();
+            string deathReason = deathReasonService.GetDeathReasonSummary(lastEvent, Core.Localization.Language.Ru);
+
+            Console.WriteLine($"Death Reason: {deathReason}");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this logging statement.](https://app.codacy.com/manual/kreghek/Zilon_Roguelike/pullRequest?prid=5045329)